src/css/styles.scss

```diff
@@ -0,0 +1,147 @@
+@import 'base';
+
+// --- deskTop ---
+
+// --- header ---
+body {
+  background-color: $primary-color;
+}
+
+header {
+  position: absolute;
+  top: 0;
+  display: flex;
+  background-color: $secondary-color;
+  width: 100%;
+  height: 15vh;
+}
+
+.user-title {
+  width: 30%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.page-title {
+  font-size: 1.5em;
+  margin-left: 4%;
+  width: 70%;
+  display: flex;
+  justify-content: flex-start;
+}
+
+//  --- main/cards
+
+.user-dashboard{
+  margin-top: 14%;
+  display: flex;
+  justify-content: space-around;
+}
+
+.card {
+  @include card(space-around, column, 50vh, 40%);
+  background-color: $card-color;
+  box-shadow: 2px 10px 10px 2px $cardShadow-color;
+  margin: 10%, 5%, 5%, 5%;
+}
+
+.search-room {
+  align-items: center;
+
+}
+
+.user-history {
+  text-align: center;
+}
+
+.date-input {
+  width: 100%;
+}
+
+.booking-submit {
+  font-size: 1em;
+  box-shadow: 2px 5px 5px 2px $cardShadow-color;
+  background-color: $secondary-color;
+  @include button(2% 15% 2% 15%, 0);
+}
+
+//  ---- bottom styling ----
+
+.footer {
+  position: fixed;
+  bottom: 0;
+  background-color: $secondary-color;
+  width: 100%;
+  height: 12vh;
+}
+// tablet
+@media screen and (max-width: 925px) {
+
+  .page-title {
+    font-size: 1.25em;
+    margin-left: 2%;
+  }
+
+}
+
+
+// ---- phone ----
+@media screen and (max-width: 414px) {
+
+  header {
+    position: relative;
+    top: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: $secondary-color;
+    width: 100%;
+    height: 10vh;
+  }
+
+  h1 {
+    margin: 0;
+    padding: 0;
+  }
+
+  h2 {
+    margin: 0;
+    padding: 0;
+  }
+
+  .page-title {
+    font-size: 1em;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+  }
+
+  .page-title {
+    font-size: .75em;
+    justify-content: center;
+    text-align: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  .user-dashboard{
+    margin-top: 5%;
+    flex-direction: column;
+  }
+
+  .card {
+    @include card(space-around, column, 80vh, 100%);
+    background-color: $card-color;
+    box-shadow: 2px 10px 10px 2px $cardShadow-color;
+    margin: 20%, 2%, 20%, 2%;
+  }
+
+}
+
+//  ---- utility stuff ----
+
+.hidden {
+  display: none;
+}
```

Comment:
Really good organization here. I noticed that you are importing base to this file. In scss I think partials, should start with an underscore so maybe change it to _base.scss. I would also make sure that webpack is compiling from whatever root file you decide on. I was having issues with it compiling correctly so Im importing all my partials to base.